nuxt.config.js

```
@@ -76,18 +83,34 @@ export default {
     '@nuxtjs/axios',
     '@nuxtjs/apollo',
     '@nuxtjs/auth-next',
+    [
+      'nuxt-gmaps',
+      {
+        key: process.env.GOOGLE_API_KEY,
```

Also add code to fallback to the hardcoded key

---

Resolved